app/controllers/checkouts_controller.rb

```diff
@@ -3,13 +3,26 @@ class CheckoutsController < ApplicationController
 
   def create
     @book = Book.find(params[:book_id])
-    checkout = Checkout.create(book: @book, user: current_user)
+    if CheckOutBook.process(book: @book, user: current_user)
+      flash[:success] = t('.successful_message')
+      redirect_to root_path
+    else
+      flash[:error] = t('.unsuccessful_message')
```

Comment:
hi @derekprior, the checkouts controller does not have any associated views. All creating a checkout (i.e. checking out a book from the library does) does is create a record in the database, mark the book as unavailable, and display that book on your profile where it can be renewed or returned.